libs/data-access/src/lib/concept/api/import-rdf-concepts.ts

```diff
@@ -0,0 +1,25 @@
+'use server';
+
+export const importRdfConcepts = async (
+  fileContent: string,
+  contentType: string,
+  catalogId: string,
+  accessToken: string
+) => {
+  console.log("Base uri:", process.env.CONCEPT_CATALOG_BASE_URI);
+  const resource = `${process.env.CONCEPT_CATALOG_BASE_URI}/import/${catalogId}`;
+  console.log("Resource:", resource);
```

Comment:
Ikke logg miljøvariabler
```suggestion
  const resource = `${process.env.CONCEPT_CATALOG_BASE_URI}/import/${catalogId}`;
```